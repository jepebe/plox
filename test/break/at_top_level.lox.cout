[38;5;1m[RuntimeError at line 1] Undefined variable 'break'.[0m
