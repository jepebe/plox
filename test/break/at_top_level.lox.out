[38;5;1m[line 1] Error at 'break': 'break' outside loop.[0m
1 error(s) and 0 warning(s) occurred
